filesystems/drs/src/main/scala/cromwell/filesystems/drs/DrsPathBuilder.scala

```diff
@@ -15,15 +15,12 @@ case class DrsPathBuilder(fileSystemProvider: DrsCloudNioFileSystemProvider) ext
 
   override def name: String = "DRS"
 
+  //DRS or DOS files can actually have no host, and treat that as valid (for now?)
```

Comment:
TOL: Is this comment helpful here? TBH I'm not sure I'd know what to do with this information here.